config/desktop-development.json

```diff
@@ -24,7 +24,7 @@
 		"automated-transfer": true,
 		"blogger-plan": false,
 		"calypsoify/gutenberg": false,
-		"code-splitting": false,
+		"code-splitting": true,
```

Comment:
Can we remove the flag or is it still used anywhere?

Reply:
still used in boot/common in a very odd way, and `config/test.json` is still setting it to false, presumably for the babel config.